geoext.ux/ux/SimplePrint/lib/GeoExt.ux/SimplePrint.js

```diff
@@ -472,16 +477,12 @@ GeoExt.ux.SimplePrint = Ext.extend(Ext.form.FormPanel, {
                         self.printProvider.download(
                             self.printProvider.getDownloadURL(job)
                         );
-                        self.progressPanel.remove(statusComponent);
-                        statusComponent.destroy();
-                        statusComponent = null;
-                        self.progressPanel.doLayout();
-                        self.progressPanel.setHeight(35 * self.progressPanel.items.length);
-                        self.doLayout();
                         return false;
                     };
-                }
-                else {
+                    self.printProvider.download(
+                        self.printProvider.getDownloadURL(job)
+                    );
```

Comment:
Yes, removing the Download link is what we want.
As for the fact to cancel the previous job when clicked a second time on Print, I would prefer to launch two jobs. @sbrunner told me it wasn't hard technically, so I think it's the best: direct download AND possible to launch more than one prints/downloads.